wp-content/plugins/tapestry/controller/class.tapestry-permissions.php

```diff
@@ -0,0 +1,14 @@
+<?php
+/**
+ * Tapestry Endpoint Permissions
+ *
+ */
+class TapestryPermissions {
+    static function postTapestryPermissions($request) {
```

Comment:
Since you are already in the `TapestryPermissions` class, I don't think it's needed to use the word `Permissions` in the name of your method here.